ghost/i18n/locales/hi/portal.json

```diff
@@ -0,0 +1,60 @@
+    "Delete account": "khaata hata do",
+    "Don't have an account?": "khaata nahin hai?",
+    "Email": "eemel",
+    "Email preference updated.": "eemel vareeyata apadet kee gaee.",
+    "Email preferences": "eemel praathamikataen",
+    "Emails": "eemel praathamikataen",
+    "Emails disabled": "eemel aksham",
+    "Get help": "madad len",
+    "Get notified when someone replies to your comment": "jab koee aapakee tippanee ka javaab de to soochana praapt karen",
+    "Give feedback on this post": "is post par pratikriya den",
+    "Less like this": "is tarah kam",
+    "Manage": "prabandhit karana",
+    "Monthly": "maheene ke",
+    "More like this": "is tarah se adhik",
+    "Name": "naam",
+    "Not receiving emails?": "eemel praapt nahin ho rahe hain?",
+    "Now check your email!": "Now check your email!",
+    "Price": "keemat",
+    "Re-enable emails": "eemel pun: saksham karen",
+    "Retry": "pun: prayaas karen",
+    "Save": "bachaana",
+    "Sending login link...": "login link bheja ja raha hai...",
+    "Sending...": "bhej rahe hein...",
+    "Sign in": "daakhil karana",
+    "Sign up": "sain ap karen",
+    "Start {{amount}}-day free trial": "{{amount}}-din ka nihshulk pareekshan praarambh karen",
+    "Submit feedback": "pratipushti den",
+    "Successfully unsubscribed": "saphalataapoorvak anasabsakraib kiya gaya",
+    "Thanks for the feedback!": "pratikriya ke lie dhanyavaad!",
+    "That didn't go to plan": "vah yojana par nahin gaya",
+    "This site is invite-only, contact the owner for access.": "yah sait keval-aamantrit hai, pahunch ke lie svaamee se sampark karen.",
+    "To complete signup, click the confirmation link in your inbox. If it doesn't arrive within 3 minutes, check your spam folder!": "sainap poora karane ke lie, apane inaboks mein pushtikaran link par klik karen. yadi yah 3 minat ke bheetar nahin aata hai, to apana spaim foldar jaanchen!",
+    "Unsubscribe from all emails": "sabhee eemel se sadasyata samaapt karen",
+    "Unsubscribing from emails will not cancel your paid subscription to {{title}}": "eemel se sadasyata samaapt karane se {{title}} ke lie aapakee sashulk sadasyata radd nahin hogee",
+    "Update your preferences": "apanee praathamikataen apadet karen",
+    "We couldn't unsubscribe you as the email address was not found. Please contact the site owner.": "eemel pata nahin milane ke kaaran ham aapakee sadasyata samaapt nahin kar sake. krpaya sait ke maalik se sampark karen.",
+    "Yearly": "saalaana",
+    "You have been successfully resubscribed": "aapako saphalataapoorvak phir se sabsakraib kar liya gaya hai",
+    "You're not receiving emails because you either marked a recent message as spam, or because messages could not be delivered to your provided email address.": "aap eemel praapt nahin kar rahe hain kyonki ya to aapane haal hee ke ek sandesh ko spaim ke roop mein chihnit kiya hai, ya kyonki sandesh aapake pradaan kie gae eemel pate par dileevar nahin kie ja sake.",
```

Comment:
"aap ~~eemel~~ email praapt nahin kar rahe hain kyonki ya to aapane haal hee ke ek sandesh ko ~~spaim~~ spam ke roop mein chihnit kiya hai, ya kyonki sandesh aapake pradaan kie gae ~~eemel~~ email pate par ~~dileevar nahin kie ja sake.~~ deliver nhi ho pa rhe hai"